File: Python/Task16/n33486.py
# https://inf-ege.sdamgia.ru/problem?id=33486

def F(n):
	if n == 0:
		return 0
	elif n % 3 == 0:
		return n + F(n - 3)
	else:
		return n + F(n - (n % 3))

print(F(26))

File: C++/Task16/n33486/n33486.cpp
// https://inf.reshuege.ru/problem?id=33486

#include <iostream>

int F(int n)
{
	if (n == 0)
	{
		return 0;
	}
	else if (n % 3 == 0)
	{
		return n + F(n - 3);
	}
	else // if(n % 3 != 0)
	{
		return n + F(n - (n % 3));
	}
}

int main()
{
	int r = F(26);
	std::cout << r << std::endl;
	return 0;
}